src/context/json.ts

```diff
@@ -1,11 +1,17 @@
 import { ResponseTransformer } from '../response'
 
+type JsonArray = (JsonValue | JsonRecord)[]
+type JsonRecord = Record<string, unknown>
+type JsonValue = string | boolean | number | JsonRecord | JsonArray | null
+
 /**
  * Sets the given Object as the JSON body of the response.
  * @example
  * res(json({ foo: 'bar' }))
+ * res(json('Some string'))
+ * res(json([1, '2', false, { ok: true }]))
  */
-export const json = (body: Record<string, any>): ResponseTransformer => {
+export const json = (body: JsonValue): ResponseTransformer => {
```

Comment:
Based on our discussion, let's annotate it as `body: any`, similar to `JSON.stringify` argument.